src/inline-decorator.js

```diff
@@ -116,12 +116,12 @@ module.exports = function InlineDecorator(wnd) {
     }
   };
 
-  // todo: links
   var supportedAttributes = {     // defaults (unused for now?)
     'font-weight'      : 'normal',
     'font-style'       : 'normal',
     'text-decoration'  : 'none',
-    'color'            : 'inherit'
+    'color'            : 'inherit',
+    'href'             : null
```

Comment:
The other currently supported attributes will always have a value for each element. Href, however, only exists as an attribute when a link has been attached to the element. The default behaviour of the inline-decorator is to create an empty array and attach values as it finds it, for href if it doesn't find one then the array remains empty.

The toss-up, as I see it, is between explicitly assigning an empty string as the value if the href attribute array is empty or just assigning null as the default when running tests.